narwhals/expr.py

```diff
@@ -1792,10 +1691,7 @@ def null_count(self: Self) -> Self:
             └──────────────────┘
         """
         return self.__class__(
-            lambda plx: self._to_compliant_expr(plx).null_count(),
-            self._is_order_dependent,
-            changes_length=self._changes_length,
-            aggregates=self._aggregates,
```

Comment:
this one wasn't right to begin with, `null_count` aggregates

will fix tomorrow